client/src/component/Search.js

```diff
@@ -0,0 +1,45 @@
+import "./User.css";
+import { useContext } from "react";
+import { AppContext } from "../App";
```

Comment:
Be carful creating PRs using files that dont exist in the main branch yet. If this was merged it would break the app. But I know it is difficult when everybody wants to work quickly

Reply:
thank you I wont merge it until the other pull request will be merged, I thought when I haven't call search componenet in any file yet it wouldn't break the codes